### ISSUE TITLE
custom feeds: let me pin a custom feed as the first, default one

### ISSUE THREAD

**Issue by aliceisjustplaying** [bluesky-social/social-app#799]
**Is your feature request related to a problem? Please describe.**

I was talking to a friend and we both realized one of the problems with the current chronological "Following" timeline is that it moves too fast ("it's like reading the firehose"), and especially with threading and replies still not working the way many think it supposed to work, it's often not a great experience to read. 

However, if I could open the app and it'd default to a custom feed, that'd change the experience a lot, for the better.

**Describe the solution you'd like**

I want to be able to pin a custom feed as my first, default one. The "Following" feed should not be treated special in that regard, and I should be able to even unpin it entirely.

**Comment by mschwendener** [comment]
I agree. I'd like to replace the default 'Following' feed with https://bsky.app/profile/did:plc:tenurhgjptubkk5zf5qhi3og/feed/only-posts and would unpin 'Following' if I could, and otherwise move it to the end of my list.

**Comment by mary-ext** [comment]
https://github.com/bluesky-social/social-app/pull/2775 should make it such that newly opened web pages would load into your last-selected feed, it doesn't let you reorder following though unfortunately.

**Comment by gaearon** [comment]
We plan to add reordering for Following
